init/modules/EESSI/2023.06.lua

```diff
@@ -42,14 +42,27 @@ function archdetect_cpu()
     end
     LmodError("Software directory check for the detected architecture failed")
 end
+function archdetect_accel()
+    local script = pathJoin(eessi_prefix, 'init', 'lmod_eessi_archdetect_wrapper_accel.sh')
```

Comment:
Chicken and egg here for https://github.com/EESSI/software-layer/actions/runs/11295312157/job/31417735905?pr=781, I guess I will put that in a separate PR

Reply:
Opened https://github.com/EESSI/software-layer/pull/783 to address this